### PR TITLE
applyToEachFace(..) - simplified each(..) for faces

### PR DESCRIPTION
**Pull request by fedorkotov**
This plugin simplifies using `Workplane.each(..)` on faces. To use each you have to select workplane coordinate system for each face before building your geometry. `Workplane.applyToEachFace()` function provided by this plugin separates tasks of choosing face coordinate system and actually building new geometry and provides a few built in ways of choosing coordinate system that 
are good enough in many cases.

See also https://github.com/CadQuery/cadquery/issues/713

### CODE REVIEW

**Comment by jmwright** [comment]
@fedorkotov Is this ready for review?

**Comment by fedorkotov** [comment]
@jmwright Yes, it is ready

**Comment by fedorkotov** [comment]
Black found no issues but the action fails with
```
 Error: Error: Received status code 403
 Error: Error trying to create GitHub check for Black: Received status code 403
```

**Review by jmwright — COMMENTED** [review]
The apply-to-each-face submodule seems to be a dead end. If I try to update the submodules for your branch, git refuses to do it.

**Comment by fedorkotov** [comment]
Sorry. Can't remember how and why did I add this `src/apply-to-each-face` thing.

**Comment by marcus7070** [comment]
I'll start my review today.

**Review by marcus7070 — CHANGES_REQUESTED** [review]
Aside from my comment about reloading the module, this all looks great to me!

File: plugins/apply_to_each_face/README.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+This plugin has no dependencies other than the cadquery library.
+
+## Usage
+
+To use this plugin after it has been installed, 
+just import it and use `applyToEachFace(..)`
+method of `Workplane`. To use this plugin in `CQ-editor` 
+you have to reload `apply_to_each_face` on each run 
+(see examples below). Otherwise second and subsequent runs will fail.
```
[the remark this answers is not in the document]
Comment (fedorkotov) [reply plugins/apply_to_each_face/README.md]: You are correct. I did not know about this CQ-Editor parameter. I modified README to include both variants (unchecking CQ-Editor parameter as a preferred option or reloading the module)

**Review by marcus7070 — APPROVED** [review]

**Comment by jmwright** [comment]
@fedorkotov Thanks!

**Pull request merged**